src/posts/2020-08-20-an-indepth-guide-to-aria-roles.md

```diff
@@ -0,0 +1,190 @@
+
+## Usage
+
+ARIA roles are added to elements using the `role="<ROLE_TYPE>` attribute, where `ROLE_TYPE` represents the specific role being added. Once an ARIA role is set on an element, it should not be changed, regardless of the state of the element. In some cases, roles have to be paired with the appropriate ARIA state or property. However, this article's examples have reduced markup to limit the scope to just ARIA roles.
+
+ARIA roles are broken down into six categories:
+
+1. Abstract Roles
+2. Document Structure Roles
+3. Landmark Roles
+4. Live Region Roles
+5. Widget Roles
+6. Window Roles
+
+### Abstract roles
+
+Abstract roles are only intended for use by browsers to help organize and streamline a document. They should not be used by developers writing HTML markup. Doing so will not result in any meaningful information being conveyed to assistive technologies or to users.
+
+### Document Structure roles
+
+Document roles are used to provide a structural description for a section of content. A majority of the available roles in this category resemble available semantic HTML tags and it is recommended to only use them when an appropriate native tag is not available. The roles in this category are also not typically interactive in nature, but may be used to contain other roles that are interactive. The `toolbar"` and `tooltip"` roles, demonstrated below, serve as good examples of when to use Document Structure roles to provide information to assistive technologies when the equivalent native HTML tag is not available.
+
+#### Toolbar example
+
+```html
+<div role="toolbar">
+  <div class="text-characteristics">
+    <button>Bold</button>
+    <button>Italic</button>
+    <button>Underline</button>
+  </div>
+
+  <div class="text-alignment">
+    <button>Left</button>
+    <button>Middle</button>
+    <button>Right</button>
+  </div>
+</div>
+```
+
+#### Tooltip example
+
+```html
+<button aria-describedby="notifications-desc">Notifications</button>  
+<div role="tooltip" id="notifications-desc">View and manage notifications</div>  
+```
+
+- View the [full list of available Document Structure roles](https://www.w3.org/TR/wai-aria/#document_structure_roles).
+
+### Landmark Roles
+
+Landmark roles identify content areas within a page, and as the name implies serve as navigational landmarks for the types of content areas that users may be interested in. These content areas include the page's primary content, collections of navigation links, and search functions to name a few.
```

Comment:
>Landmark roles identify content areas within a page, and as the name implies serve as navigational landmarks for the types of content areas that users may be interested in.

How about this?

>Landmark roles identify content areas within a page. They serve as navigational landmarks for the types of content areas that users may be interested in.

My thinking is that these are two separate thoughts, and it helps to keep the sentences shorter.